### ISSUE TITLE
Add prefix `v1` to tracker API endpoint

### ISSUE THREAD

**Issue by josecelano** [torrust/torrust-index-backend#94]
The tracker has added a prefix to the API endpoints with the version:

https://github.com/torrust/torrust-tracker/pull/239

So API URLs have changed from (without version):

`http://0.0.0.0:1212/api/stats?token=MyAccessToken`

To (with version on the path `v1`):

`http://0.0.0.0:1212/api/v1/stats?token=MyAccessToken`

We have to add the prefix in the [TrackerService](https://github.com/torrust/torrust-index-backend/blob/develop/src/tracker.rs).

For the time being, the tracker also keeps the old URLs. Once this issue is finished, we have to open a new issue on the [tracker repo](https://github.com/torrust/torrust-tracker) to remove the support for API URLs without the version prefix.

**Comment by josecelano** [comment]
Hi @da2ce7 @WarmBeer I think in the long term, we could extract the [API client](https://github.com/torrust/torrust-tracker/blob/develop/tests/servers/api/v1/client.rs) into an independent package or as a feature in an API package. But for the time being, I'm going to just add the prefix `v1`.
